site/content/events/2016-denver/_sponsors.txt

```diff
@@ -237,6 +237,13 @@ filter: erb
     :width  => '150px',
     :height => '50px'
 },
+{
+    :name   => 'Confrence Ops',
```

Comment:
Updated, thanks for the catch!


On Sat, 16 Apr 2016 at 19:25 Bridget Kromhout <notifications@github.com>
wrote:

> In site/content/events/2016-denver/_sponsors.txt
> <https://github.com/jedi4ever/devopsdays-webby/pull/2711#discussion_r59976797>
> :
>
> > @@ -237,6 +237,13 @@ filter: erb
> >      :width  => '150px',
> >      :height => '50px'
> >  },
> > +{
> > +    :name   => 'Confrence Ops',
>
> Spelling?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jedi4ever/devopsdays-webby/pull/2711/files/d91bd8259506db3ed2b81b79a23b20bd6660027c#r59976797>
>